AURN_Data_Download/AURN_download.py

```diff
@@ -598,11 +598,10 @@ def test_preprocess_code(df_in,pt,spc_zero_process = ['O3','NO2','NOXasNO2'],min
     parser = argparse.ArgumentParser(description="*** A script for automated downloading of AURN data for a given date range. ***")
     parser.add_argument("--meta_data_url", "-m", help="url of the AURN metadata")
     parser.add_argument("--meta_data_filename", "-f", help="filename of the AURN metadata in RData format (.RData). \
-                                                           Default: " + DEFAULT_METADATA_FILE)
+                                                           Default: {}".format(DEFAULT_METADATA_FILE))
     parser.add_argument("--emep_filename","-e", default=None, help="filename of the emep file in CSV format (.csv)")
     parser.add_argument("--years", "-y", metavar='Y', type=int, nargs='+', help="the years to be processed. Must be \
-        in (and defaults to) {}".format(
-        '[' + ", ".join([str(int) for int in AVAILABLE_YEARS]) + ']'))
+        in (and defaults to) {}".format('[' + ", ".join([str(int) for int in AVAILABLE_YEARS]) + ']'))
```

Comment:
it's a little frustrating that we need to mix .format() and .join - but I've had a look around and can't see anything more appropriate than what you've done here, so let's stick with it